pkg/gardenlet/controller/factory.go

```diff
@@ -218,14 +218,15 @@ func (f *GardenletControllerFactory) Run(ctx context.Context) error {
 
 	go extensionsController.Run(controllerCtx, *f.cfg.Controllers.ControllerInstallationRequired.ConcurrentSyncs, *f.cfg.Controllers.ShootStateSync.ConcurrentSyncs)
 
-	logger.Logger.Infof("Gardenlet (version %s) initialized.", version.Get().GitVersion)
+	//nolint
```

Comment:
Instead of adding `//nolint` here, we could extend https://github.com/gardener/gardener/blob/21b1b1f3dd40936a76ed54a8d200ad55f85a19a8/.golangci.yaml#L41-L42